modules/javafx.controls/src/main/java/javafx/scene/control/SpinnerValueFactory.java

```diff
@@ -409,8 +413,13 @@ private T _getValue(int index) {
     /**
      * A {@link javafx.scene.control.SpinnerValueFactory} implementation designed to iterate through
      * integer values.
-     *
-     * <p>Note that the default {@link #converterProperty() converter} is implemented
+     * <p>
+     * If {@link #wrapAround} is {@code true}, the {@code IntegerSpinnerValueFactory} will step from
```

Comment:
The link for wrapAround does not resolve in Eclipse.
I'd suggest to change it to
```
* If {@link SpinnerValueFactory#wrapAround wrapAround}
```

Same comment applies to the min and max in line 420.

Reply:
I don't think this will help. This is the same issue I mentioned in PR #1389 -- you need to link to the property _method_, so I think changing to the following will do it:

```
* If {@link #wrapAroundProperty wrapAround}
```

(it shouldn't be necessary to qualify it with the enclosing class, but it wouldn't hurt to do so)

Reply:
`{@link #wrapAroundProperty wrapAround}` does not resolve **in Eclipse** (it does in javadoc output).

This might be Eclipse limitation, but it would be awfully nice to have it resolved in Eclipse.

Reply:
I also fixed the min/max links to refer to minProperty/maxProperty. IntelliJ IDEA has the same problem as eclipse. I guess this has to do with the "javafx" option to the javadoc command (build.gradle:4286 `options.addBooleanOption("javafx").setValue(true);`) that seems to allow to use the private fields' docs for the properties, by naming convention? Maybe we need to file issues in Eclipse/IntelliJ for this so they add the option to their javadoc renderer.

Reply:
There are two issues here. The first is that you cannot just link to a `xxx` property by name -- you have to link to the `xxxProperty` method. The former doesn't work in the generated javadocs either (so it's not just an IDE problem). The second issue seems specific to Eclipse, in that if you don't prefix the method name with the name of the enclosing (outer) class, Eclipse doesn't generate the link. That seems like an Eclipse bug to me, but as Andy points out, it's a trivial thing to add the class name (and won't affect the output).